hedera-node/hapi-utils/src/main/java/com/hedera/node/app/hapi/utils/CommonUtils.java

```diff
@@ -205,7 +205,7 @@ public static HederaFunctionality functionOf(@NonNull final TransactionBody txn)
     }
 
     /**
-     *get the EVM address from the long number
+     * get the EVM address from the long number.
```

Comment:
Putting a period at the end of comments is a really annoying convention since this comment is not actually a complete sentence. Just an opinion

Reply:
This follows the Oracle Javadoc standards as well as our select Google Code Style: https://google.github.io/styleguide/javaguide.html#s7.2-summary-fragment

Reply:
😭 ok, thanks 👍